src/runtime/node/inspector/index.ts

```diff
@@ -0,0 +1,34 @@
+// https://nodejs.org/api/inspector.html
+import noop from "../../mock/noop";
+import mock from "../../mock/proxy";
+import type inspector from "node:inspector";
+import type Console from "node:console";
+
+export const close: typeof inspector.close = noop;
+
+export const console: Console = mock.__createMock__("inspector.console");
+
+export const open: typeof inspector.open = () => ({
+  __unenv__: true,
+  [Symbol.dispose]() {
+    return Promise.resolve();
+  },
+});
+
+export const url: typeof inspector.url = () => undefined;
+
+export const waitForDebugger: typeof inspector.waitForDebugger = noop;
+
+// `node:inspector` and `node:inspector/promises` share the same implementation with only Session being in the promises module:
+// https://github.com/nodejs/node/blob/main/lib/inspector/promises.js
+export const Session: typeof inspector.Session =
+  mock.__createMock__("inspector.Session");
```

Comment:
`Session.post`'s promise won't be resolved (it is not only limited to here. made tracker: https://github.com/unjs/unenv/issues/212)